config/install/field.field.media.instagram.field_instagram_shortcode.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_instagram_shortcode
+    - media_entity.bundle.instagram
+id: media.instagram.field_instagram_shortcode
+field_name: field_instagram_shortcode
+entity_type: media
+bundle: instagram
+label: 'Instagram Shortcode'
+description: ''
```

Comment:
It is still valuable for site builders. Remember, this module is also an example of how things should be implemented. Having nice descriptions will help us to reach that goal as they are still visible in the config UI.